tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/Entities/SolutionEntitySteps.cs

```diff
@@ -132,7 +132,7 @@ await SolutionEntityBuilder.Create()
             }
         }
 
-        [Then(@"Solution have following details")]
+        [Then(@"solutions have the following details")]
         public static async Task ThenSolutionDetailExist(Table table)
```

Comment:
Let's rename this method so that it matches the text, e.g. `ThenSolutionsHaveTheFollowingDetails`.

Reply:
Method has been renamed to `ThenSolutionsHaveTheFollowingDetails`